packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -0,0 +1,33 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+import BigNumber from 'bignumber.js'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+/**
+ * CreateLoanScheme DeFi Transaction
+ */
+export interface CreateLoanScheme {
+  ratio: number
+  rate: BigNumber
+  identifier: string
+  update: bigint
+}
+
+/**
+ * Composable CreateLoanScheme, C stands for Composable.
+ * Immutable by design, bi-directional fromBuffer, toBuffer deep composer.
+ */
+export class CCreateLoanScheme extends ComposableBuffer<CreateLoanScheme> {
+  static OP_CODE = 0x4c
+  static OP_NAME = 'OP_DEFI_TX_CREATE_LOAN_SCHEME'
+
+  composers (cls: CreateLoanScheme): BufferComposer[] {
+    return [
+      ComposableBuffer.uInt32(() => cls.ratio, v => cls.ratio = v),
+      ComposableBuffer.satoshiAsBigNumber(() => cls.rate, v => cls.rate = v),
+      ComposableBuffer.varUIntUtf8BE(() => cls.identifier, v => cls.identifier = v),
+      ComposableBuffer.int64(() => cls.update, v => cls.update = v)
```

Comment:
For this one, the c++ side type is `uint64_t`, so ideally we also should consider the `unsigned` part. The `update` member variable is used to indicate the `ACTIVATE_AFTER_BLOCK` property.
may be try `bigNumberUInt64`